pipeline-scripts/release.groovy

```diff
@@ -10,48 +10,50 @@ def stageVersions() {
     sh "elliott --version"
 }
 
-def stageValidation(quay_url) {
+def stageValidation(quay_url, name, advisory) {
     echo "Verifying payload does not already exist"
     res = commonlib.shell(
             returnAll: true,
-            script: "GOTRACEBACK=all ${oc_cmd} adm release info ${quay_url}:${params.NAME}"
+            script: "GOTRACEBACK=all ${oc_cmd} adm release info ${quay_url}:${name}"
     )
 
     if(res.returnStatus == 0){
-        error("Payload ${params.NAME} already exists! Cannot continue.")
+        error("Payload ${name} already exists! Cannot continue.")
     }
 
     // AMH - This may be optional?
-    if (params.ADVISORY != "") {
+    if (advisory != "") {
         echo "Verifying advisory exists"
         res = commonlib.shell(
                 returnAll: true,
-                script: "elliott get ${params.ADVISORY}"
+                script: "elliott get ${advisory}"
         )
 
         if(res.returnStatus != 0){
-            error("Advisory ${params.ADVISORY} does not exist! Cannot continue.")
+            error("Advisory ${advisory} does not exist! Cannot continue.")
         }
     }
 }
 
-def stageGenPayload(quay_url) {
+def stageGenPayload(quay_url, name, from_release_tag, description, previous, errata_url) {
     // build metadata blob
-    def metadata = "{\"description\": \"${params.DESCRIPTION}\""
-    if (params.ERRATA_URL != "") {
-        metadata += ", \"url\": \"${params.ERRATA_URL}\""
+    def metadata = "{\"description\": \"${description}\""
+    if (errata_url != "") {
+        metadata += ", \"url\": \"${errata_url}\""
     }
     metadata += "}"
 
     // build oc command
     def cmd = "GOTRACEBACK=all ${oc_cmd} adm release new "
-    cmd += "--from-release=registry.svc.ci.openshift.org/ocp/release:${params.FROM_RELEASE_TAG} "
-    if (params.PREVIOUS != "") {
-        cmd += "--previous \"${params.PREVIOUS}\" "
+    cmd += "--from-release=registry.svc.ci.openshift.org/ocp/release:${from_release_tag} "
+    if (previous != "") {
+        cmd += "--previous \"${previous}\" "
     }
-    cmd += "--name ${params.NAME} "
-    cmd += "--metadata '${metadata}' "
-    cmd += "--to-image=${quay_url}:${params.NAME} "
+    cmd += "--name ${name} "
+    if (description != "") {
```

Comment:
description has only ever been empty, but we do want the errata url. actually, i don't see why this is conditional at all; it wasn't before.
```suggestion
```